ee/modules/650-runtime-audit-engine/docs/README.md

```diff
@@ -0,0 +1,107 @@
+> However, a Kubernetes cluster with the autoscaling feature enabled makes it hard to operate. 
+> Additional security mechanisms of Deckhouse (implemented by other modules), such as multitenancy and admission policy control, provide the required level of security to mitigate attacks on the Falco DaemonSet.
+
+There are five different containers in a single agent Pod:
+![Falco Pod](../../images/650-runtime-audit-engine/falco_pod.png)
+
+1. `falco-driver-loader` — this init container compiles the eBPF program and saves it in an empty dir to make it available to Falco.
+2. `falco` — collects events, enriches them with metadata and saves them.
+3. `rules-loader` — collects ([FalcoAuditRules](cr.html#falcoauditrules)) CRs from Kubernetes and saves them in a shared directory (empty dir).
+4. `falcosidekick` — exports events as metrics on which alerts can be generated.
+5. `kube-rbac-proxy` — protects the `falcosidekick` metric's endpoint.
+
+## Audit Rules
+
+The event collection itself is a low-yielding activity because the amount of data coming from the Linux kernel is too large to be analyzed by a human.
+Rules address this problem by collecting events according to certain pattens that can help in detecting suspicious activities.
+
+The main part of a rule is a conditional expression (which uses the [conditions syntax](https://falco.org/docs/rules/conditions/)).
+
+### Embedded rules
+
+There are several built-in rules that cannot be disabled.  
+These rules are aimed at detecting Deckhouse security problems as well as security problems affecting the `runtime-audit-engine` module.
+
+- `/etc/falco/falco_rules.yaml` — syscall rules;
+- `/etc/falco/k8s_audit_rules.yaml` — Kubernetes audit rules.
+
+
+### Custom audit rules
+
+Users can use a `FalcoAuditRules` CRD to add custom security audit rules. 
+Each Falco agent Pod has a sidecar container running [shell-operator](https://github.com/flant/shell-operator).
+This sidecar reads rules from the custom resources and saves them in the Pod's `/etc/falco/rules.d/` directory.
+Falco automatically reloads the configuration when a new rule becomes available.
+
+![Falco shell-operator](../../images/650-runtime-audit-engine/falco_shop.png)
+
+Such a schema allows the IaC approach to be used to maintain Falco rules.
+
+## Requirements
+
+### OS
+
+The module uses the eBPF Falco driver to ingest syscall data. It is better suited for environments where loading a kernel module is prohibited or unsupported, such as GKE, EKS, and other managed Kubernetes solutions.
+However, there are known limitations to the eBPF driver:
+* The eBPF probe may not work for every system.
+* At least a Linux kernel version 4.14 is required, although the Falco project suggests an LTS kernel version 4.14/4.19 or higher.
+
+### CPU / Memory
+
+Falco agents are running on every node. Therefore, the resource consumption of each Pod depends on the number of rules or ingested events.
```

Comment:
> depends on the number of rules or ingested events

Again, is it really OR, or, maybe, it is AND?

Reply:
Or, yes.